js/src/modals/Verification/verification.js

```diff
@@ -52,12 +55,15 @@ import SendConfirmation from './SendConfirmation';
 import Done from './Done';
 
 @observer
-export default class Verification extends Component {
+class Verification extends Component {
+  static contextTypes = {
+    api: PropTypes.object.isRequired
+  }
+
   static propTypes = {
-    store: nullableProptype(PropTypes.object.isRequired),
     account: PropTypes.string.isRequired,
-    onSelectMethod: PropTypes.func.isRequired,
-    onClose: PropTypes.func.isRequired
+    onClose: PropTypes.func.isRequired,
+    isTestnet: PropTypes.bool.isRequired
```

Comment:
Would prefer consistency. It comes in as `isTest`, so keep it as `isTest` (We are doing things in 2 different ways now, some has the one others have the other)

In addition, consider keeping alphabetical order here. (i before o)